substrate/frame/revive/src/lib.rs

```diff
@@ -359,15 +360,16 @@ pub mod pallet {
 
 			#[inject_runtime_type]
 			type RuntimeCall = ();
-			type CallFilter = ();
+
+			type CallFilter = Everything;
```

Comment:
Will be removed in https://github.com/paritytech/polkadot-sdk/pull/8584.

Let's avoid conflicts.

```suggestion
			type CallFilter = ();
```